src/glyph.ts

```diff
@@ -5,20 +5,133 @@ import { Flow } from './tables';
 import { Element } from './element';
 import { BoundingBoxComputation } from './boundingboxcomputation';
 import { BoundingBox } from './boundingbox';
+import { Font } from './smufl';
+import { RenderContext } from './types/common';
+import { Stave } from './stave';
+
+export interface KeyProps {
```

Comment:
Only `GlyphProps` belongs here. Let's leave 'KeyProps` and `TypeProps` in common for now.

Reply:
ok